script.js

```diff
@@ -40,10 +40,8 @@ class Books {
     for (let i = 0; i < this.collection.length; i += 1) {
       books.innerHTML += `  
       <div class="book-container">
-        <div class = "each-book">
-          <p class="book-title">"${this.collection[i].title}" by ${this.collection[i].author}</p> 
-          <button type="button" class="remove-button">Remove</button>
-        </div>
+        <p class="book-title">"${this.collection[i].title}" by ${this.collection[i].author}</p> 
+        <button type="button" class="remove-button">Remove</button>
       </div>
       `;
     }
```

Comment:
- [ ] [REQUIRED] Hello team, you have done a great job with the codebase in the `script.js` file however, when I add an item, it only gets added in the local storage but does not get added on the UI. Kindly ensure the added item gets displayed on the `list`.
- [ ] [REQUIRED] Please note that items been added should be removable. A good example to achieve this by using the [Element.classList](https://developer.mozilla.org/en-US/docs/Web/API/Element/classList#examples) add and remove methods.